pilot/pkg/networking/core/route/route.go

```diff
@@ -1242,6 +1242,9 @@ func BuildDefaultHTTPInboundRoute(clusterName string, operation string) *route.R
 		// gRPC requests time out like any other requests using timeout or its default.
 		GrpcTimeoutHeaderMax: Notimeout,
 	}
+	out.GetRoute().RetryPolicy = &route.RetryPolicy{
+		RetryOn: "connect-failure,reset",
```

Comment:
The "safe" retry name is reset-before-request"

Reply:
yes. Just added these as examples before your PR. Will change now